lib/core.js

```diff
@@ -435,7 +435,7 @@ const startServers = () => {
 
   let workerId = 0;
 
-  serverNames.forEach(serverName => {
+  for (const serverName in configServers) {
```

Comment:
```suggestion
  for (const serverName of configServers) {
```

Reply:
in this case: `for (const serverName of serverNames) {`